src/components/posts/PostText.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+function PostText(props) {
+  return <p style={styles.bodyTextStyles}>{props.bodyText}</p>;
+}
+export default PostText;
+
+const styles = {
+  bodyTextStyles: {
+    marginLeft: "20px",
+    marginRight: "20px",
+    marginBottom: "20px",
+    marginTop: "20px",
```

Comment:
Is there a reason you didn't use `margin: 20px` here?

Reply:
Oops, didn't know that was an option (still learning styling)